src/platform/plugins/private/kibana_overview/tsconfig.json

```diff
@@ -6,6 +6,7 @@
   "include": [
     "public/**/*",
     "common/**/*",
+    "../../../../../typings/emotion.d.ts"
```

Comment:
@eokoneyo should be an expert in this! Is it really needed? If so, I think something else might be wrong.